### PR TITLE
Change Capability API

### PR DESCRIPTION
**Pull request by leandroalonso**
This PR changes the way Capabilities are created. It aims to be simpler.

### To test

Test the Media Editor and make sure everything is working. There are no feature changes in this PR.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Sources/Capabilities/MediaEditorCapability.swift [hunk 1]
```diff
@@ -1,21 +1,28 @@
 import UIKit
 
+/// A type that is a UIViewController and also conforms to MediaEditorCapability
```
Comment (emilylaguna) [inline Sources/Capabilities/MediaEditorCapability.swift:3]: 👍Thanks for adding the documentation!

File: Sources/MediaEditor.swift [hunk 2]
```diff
@@ -12,7 +12,7 @@ import UIKit
 */
 open class MediaEditor: UINavigationController {
     /// The capabilities are displayed in the Media Editor. You can add your own capabilities here.
-    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorFilters.self, MediaEditorCropZoomRotate.self]
+    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorFiltersViewController.self, MediaEditorCropZoomRotate.self]
```
Comment (emilylaguna) [inline Sources/MediaEditor.swift]: I find it a bit odd that one of the capabilities has the 'ViewController' in the name and the other doesn't. Thoughts?

File: Example/Extending/BrightnessCapability/BrightnessCapability.swift [hunk 3]
```diff
@@ -99,3 +69,21 @@ class BrightnessViewController: UIViewController {
         return UIStoryboard(name: "BrightnessCapability", bundle: .main).instantiateViewController(withIdentifier: "brightnessViewController") as! BrightnessViewController
     }
 }
+
+extension BrightnessViewController: MediaEditorCapability {
```
Comment (emilylaguna) [inline Example/Extending/BrightnessCapability/BrightnessCapability.swift:73]: I like this, much cleaner.

**Review by emilylaguna — CHANGES_REQUESTED** [review]
Looks good, couple of comments, nothing major.

File: Sources/MediaEditor.swift (hunk 2 above)
Comment (leandroalonso) [reply Sources/MediaEditor.swift]: Renamed in 65677df328cdd048fd6dfa30b02a939db6e7c31d

**Review by emilylaguna — APPROVED** [review]
👍LGTM!

**Pull request merged**